sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -649,7 +649,7 @@ private async Task<Response<RecognizeEntitiesResultCollection>> RecognizeEntitie
                     batchInput,
```

Comment:
there are other places where `_stringCodeUnit` is specified here and were not changed